.github/workflows/rust.yml

```diff
@@ -9,28 +9,38 @@ on:
     - cron: "21 3 * * 5"
 
 jobs:
-  test-freebsd:
+  test-vm:
+    strategy:
+      matrix:
+        os: [freebsd, netbsd]
+        rust: [stable, beta]
```

Comment:
Currently this doesn't work and causes syntax errors. It seems to be because github actions don't allow `${{x}}` variables in `uses:` field...... so I'm suggesting using cross instead of vmactions, which would then allow to use something like `cross build --target ${{matrix.target}}` in `run: |` field.

Reply:
if don't use vmactions, we can't run the tests, right?

Reply:
Sorry, I didn't think of that. It's true that using cross will only compile executables for the target platform, so it's best not to use matrix?

Reply:
Yes, I guess we can't use the matrix here -- sorry for the misdirection!